webtau-core/src/main/java/com/twosigma/webtau/Matchers.java

```diff
@@ -0,0 +1,268 @@
+     */
+    public static ActualCodeExpectations code(CodeBlock codeBlock) {
+        return new ActualCode(codeBlock);
+    }
+
+    /**
+     * Equal matcher
+     * <pre>
+     * actual(value).should(equal(10));
+     * </pre>
+     * @param expected value to be equal to
+     * @return matcher instance
+     */
+    public static EqualMatcher equal(Object expected) {
+        return new EqualMatcher(expected);
+    }
+
+    /**
+     * Not equal matcher
+     * <pre>
+     * actual(value).should(notEqual(10));
+     * </pre>
+     * @param expected value to not be equal to
+     * @return matcher instance
+     */
+    public static NotEqualMatcher notEqual(Object expected) {
+        return new NotEqualMatcher(expected);
+    }
+
+    /**
+     * Contain matcher
+     * <pre>
+     * actual(value).should(contain(10));
+     * </pre>
+     * @param expected value to be contained
+     * @return matcher instance
+     */
+    public static ContainMatcher contain(Object expected) {
+        return new ContainMatcher(expected);
+    }
+
+    /**
+     * Contain matcher. Alias to contain
+     * <pre>
+     * actual(value).should(containing(10));
+     * </pre>
+     * @param expected value to be contained
+     * @return matcher instance
+     */
+    public static ContainMatcher containing(Object expected) {
+        return new ContainMatcher(expected);
+    }
+
+    /**
+     * Greater than matcher
+     * <pre>
+     * actual(value).shouldBe(greaterThan(10));
+     * </pre>
+     * @param expected value to be greater than
+     * @return matcher instance
+     */
+    public static GreaterThanMatcher greaterThan(Object expected) {
+        return new GreaterThanMatcher(expected);
+    }
+
+    /**
+     * Greater than or equal matcher
+     * <pre>
+     * actual(value).shouldBe(greaterThanOrEqual(10));
+     * </pre>
+     * @param expected value to be greater than or equal
+     * @return matcher instance
+     */
+    public static GreaterThanOrEqualMatcher greaterThanOrEqual(Object expected) {
+        return new GreaterThanOrEqualMatcher(expected);
+    }
+
+    /**
+     * Less than matcher
+     * <pre>
+     * actual(value).shouldBe(lessThan(10));
+     * </pre>
+     * @param expected value to be less than
+     * @return matcher instance
+     */
+    public static LessThanMatcher lessThan(Object expected) {
+        return new LessThanMatcher(expected);
+    }
+
+    /**
+     * Less than or equal matcher
+     * <pre>
+     * actual(value).shouldBe(lessThanOrEqual(10));
+     * </pre>
+     * @param expected value to be less than
+     * @return matcher instance
+     */
+    public static LessThanOrEqualMatcher lessThanOrEqual(Object expected) {
+        return new LessThanOrEqualMatcher(expected);
+    }
+
+    /**
+     * Throw exception <code>code</code> matcher.
+     * <pre>
+     * code(() -> {
+     *     businessLogic(-10);
+     * }).should(throwException("negative are not allowed"));
```

Comment:
This should be either `negatives are not allowed` or better still `negative numbers`.

Reply:
sounds good, thank you